services/pom.xml

```diff
@@ -36,6 +36,9 @@
         <maven.compiler.release>17</maven.compiler.release>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
+        <mockito.version>1.9.5</mockito.version>
+        <mockito.junit-jupiter.version>5.0.0</mockito.junit-jupiter.version>
+        <junit.jupiter.version>5.9.2</junit.jupiter.version>
```

Comment:
We already use this `junit.jupiter` version in some other module for quarkus testing. Is it possible to extract this into the project's parent pom and add it to the dependencies management, so that all our java modules use same version?

At the same time, I am not 100% sure if this is needed if you move the tests to `integration-arquillian` as I've suggested above?

Reply:
Moved it to the dependency management in [9cb295c](https://github.com/keycloak/keycloak/pull/26484/commits/9cb295cee525ddf88ffc5a9fdd4d624fb2e567ec)

Reply:
Cool, even if we don't use junit5 in services like I suggested in other point, it doesn't hurt to have this moved in parent `pom.xml` dependency management. Thanks!